thorlcr/activities/hashdistrib/thhashdistribslave.cpp

```diff
@@ -147,8 +147,9 @@ class CDistributorBase : public CSimpleInterface, implements IHashDistributor, i
                 OwnedConstThorRow row = dedupList.getClear(--i);
                 if ((NULL != prev.get()) && (0 == iCompare->docompare(prev, row)))
                 {
-                    size32_t rsz = owner.rowMemSize(row);
-                    total -= rsz;
+                    /* NB: do not alter 'total' size. It represents the amount originally added to the bucket
+                     * which will be deducted when sent.
+                     */
```

Comment:
NB: This change doesn't effect the sizes used. It so happens that the size used by the write handlers, was requested before dedup, so this downsizing on dedupped records was never used (or required).